src/log/log.h

```diff
@@ -1,51 +1,54 @@
 /**
-* @file log/log.h
-*
-* @brief Logging library
-*
-* This library can be used to produce logs during simulation runs.
-*
-* @copyright
-* Copyright (C) 2008-2021 HPDCS Group
-* https://hpdcs.github.io
-*
-* This file is part of ROOT-Sim (ROme OpTimistic Simulator).
-*
-* ROOT-Sim is free software; you can redistribute it and/or modify it under the
-* terms of the GNU General Public License as published by the Free Software
-* Foundation; only version 3 of the License applies.
-*
-* ROOT-Sim is distributed in the hope that it will be useful, but WITHOUT ANY
-* WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
-* A PARTICULAR PURPOSE. See the GNU General Public License for more details.
-*
-* You should have received a copy of the GNU General Public License along with
-* ROOT-Sim; if not, write to the Free Software Foundation, Inc.,
-* 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
-*/
+ * @file log/log.h
+ *
+ * @brief Logging library
+ *
+ * This library can be used to produce logs during simulation runs.
+ *
+ * SPDX-FileCopyrightText: 2008-2021 HPDCS Group <rootsim@googlegroups.com>
+ * SPDX-License-Identifier: GPL-3.0-only
+ */
 #pragma once
 
 #include <stdbool.h>
 
 #ifndef LOG_LEVEL
+/// The minimum logging level supported at compile time
+/** Compiler optimizations remove log calls with lower level than this one */
```

Comment:
Maybe this comment breaks Doxygen syntax and only the last one is considered for the documentation.

Reply:
Actually this syntax is suggested by the Doxygen documentation:

- the first line, starting with **///**, is recognized as the brief description,
- the second line, enclosed in **/**** and ***/**, is mapped to the "long" description.

This syntax works only if both sections are one line long.